miner/worker.go

```diff
@@ -1070,7 +1070,13 @@ func (w *worker) commitTransactions(env *environment, txs *types.TransactionsByP
 
 	var depsWg sync.WaitGroup
 
-	EnableMVHashMap := false
+	var EnableMVHashMap bool
+
+	if w.chainConfig.Bor.IsParallelUniverse(env.header.Number) {
+		EnableMVHashMap = true
```

Comment:
Do you think this can be simplified with 
`EnableMVHashMap = w.chainConfig.Bor.IsParallelUniverse(env.header.Number)` ?

Reply:
Yesser! Will do.

Reply:
[Done](https://github.com/maticnetwork/bor/pull/1039/commits/31994a53f134b26408ceafccdd71a813ff7570db).